richtextfx-demos/src/main/java/org/fxmisc/richtext/demo/richtext/RichText.java

```diff
@@ -130,8 +130,8 @@ public void start(Stage primaryStage) {
         textColorPicker.valueProperty().addListener((o, old, color) -> updateTextColor(color));
         backgroundColorPicker.valueProperty().addListener((o, old, color) -> updateBackgroundColor(color));
 
-        undoBtn.disableProperty().bind(Bindings.not(area.undoAvailableProperty()));
-        redoBtn.disableProperty().bind(Bindings.not(area.redoAvailableProperty()));
+        area.undoAvailableProperty().values().feedTo(undoBtn.disableProperty());
+        area.redoAvailableProperty().values().feedTo(redoBtn.disableProperty());
```

Comment:
> That would be necessary when either
>
>   - you wanted to reuse the button later for a different area (obviously); or
>   - (less obviously, and more importantly) if the life time of the area was longer than that of the button, so that the button could be garbage collected when no longer used.

Right, but my point is that since neither of the above occur, is it ok if I just do what I did since `Subscription#unsubscribe` will never be called?

> undoBtn.disableProperty().bind(area.undoAvailableProperty().map(x -> !x));

Ah, I like that approach better.